lib/minstall.js

```diff
@@ -133,315 +118,586 @@ function checkStartConditions() {
     });
 }
 
-function linkModulesToRoot(moduleInfos) {
-  logger.debug('deleting installed versions of local modules');
-  return Promise.all(moduleInfos.map((moduleInfo) => {
-    return systools.delete(path.join(cwd, 'node_modules', moduleInfo.folderName));
-  }))
-    .then(() => {
-      logger.debug('linking local modules to root-node_modules');
-      return Promise.all(moduleInfos.map((moduleInfo) => {
+function setupLogger() {
+  logger.remove(logger.transports.Console);
+  logger.add(logger.transports.Console, {
+    stderrLevels: ['warn', 'error', 'critial'],
+    colorize: true,
+    handleExceptions: true,
+    humanReadableUnhandledException: true,
+    timestamp: false,
+    prettyPrint: true,
+  });
 
-        // local modules should be linked using the folder-names they should have,
-        // no matter what folder-name they actually have, therefore don't use .realFolderName here
-        const targetPath = path.join(cwd, 'node_modules', moduleInfo.folderName);
-        const modulePath = path.join(cwd, moduletools.modulesFolder, moduleInfo.realFolderName);
-        return systools.link(modulePath, targetPath);
-      }));
+  const logLevels = {
+    critical: {level: 0, color: 'red'},
+    error: {level: 1, color: 'magenta'},
+    warn: {level: 2, color: 'yellow'},
+    info: {level: 3, color: 'green'},
+    verbose: {level: 4, color: 'gray'},
+    debug: {level: 5, color: 'blue'},
+    silly: {level: 6, color: 'cyan'},
```

Comment:
yeah I found something I do not like ;)

Reply:
do you need everyone of this level?

Please drop those you do not need.

Reply:
https://github.com/heikomat/minstall/pull/28/files#diff-aab1c68f02682c785afd35196f518d24R124 

I guess only three are needed.

Reply:
I could live with reducing the loglevels to `error`, `warn`, `info` and `debug`, but the levels you found are just the ones that log to `stderr`. The levels that are not listed there log to `stdout`